swift_package/BrainFlowCI/BrainFlowBindings/BoardShim.swift

```diff
@@ -0,0 +1,582 @@
+        var samplingRate: Int32 = 0
+        let errorCode = get_sampling_rate (boardId.rawValue, preset.rawValue, &samplingRate)
+        try checkErrorCode("Error in board info getter", errorCode)
+
+        return samplingRate
+    }
+
+    /**
+     * get row index in returned by get_board_data() 2d array which contains timestamps
+     */
+    static func getTimestampChannel (_ boardId: BoardIds, _ preset: BrainFlowPresets = .DEFAULT_PRESET) throws -> Int32 {
+        var timestampChannel: Int32 = 0
+        let errorCode = get_timestamp_channel (boardId.rawValue, preset.rawValue, &timestampChannel)
+        try checkErrorCode("Error in board info getter", errorCode)
+
+        return timestampChannel
+    }
+
+    /**
+     * get row index in returned by get_board_data() 2d array which contains markers
+     */
+    static func getMarkerChannel (_ boardId: BoardIds, _ preset: BrainFlowPresets = .DEFAULT_PRESET) throws -> Int32
+    {
+        var markerChannel: Int32 = 0
+        let errorCode = get_marker_channel (boardId.rawValue, preset.rawValue, &markerChannel)
+        try checkErrorCode("Error in board info getter", errorCode)
+
+        return markerChannel
+    }
+
+    /**
+     * get number of rows in returned by get_board_data() 2d array
+     */
+    static func getNumRows (_ boardId: BoardIds, _ preset: BrainFlowPresets = .DEFAULT_PRESET) throws -> Int32 {
+        var numRows: Int32 = 0
+        let errorCode = get_num_rows (boardId.rawValue, preset.rawValue, &numRows)
+        try checkErrorCode("Error in board info getter", errorCode)
+
+        return numRows
+    }
+
+    /**
+     * get row index in returned by get_board_data() 2d array which contains package nums
+     */
+    static func getPackageNumChannel (_ boardId: BoardIds, _ preset: BrainFlowPresets = .DEFAULT_PRESET) throws -> Int32 {
+        var pkgNumChannel: Int32 = 0
+        let errorCode = get_package_num_channel (boardId.rawValue, preset.rawValue, &pkgNumChannel)
+        try checkErrorCode("Error in board info getter", errorCode)
+
+        return pkgNumChannel
+    }
+
+    /**
+     * get row index in returned by get_board_data() 2d array which contains battery level
+     */
+    static func getBatteryChannel (_ boardId: BoardIds, _ preset: BrainFlowPresets = .DEFAULT_PRESET) throws -> Int32 {
+        var batteryChannel: Int32 = 0
+        let errorCode = get_battery_channel (boardId.rawValue, preset.rawValue, &batteryChannel)
+        try checkErrorCode("Error in board info getter", errorCode)
+
+        return batteryChannel
+    }
+
+    /**
+     * Get names of EEG electrodes in 10-20 system. Only if electrodes have freezed
+     * locations
+     */
+    static func eegNames (_ boardId: BoardIds, _ preset: BrainFlowPresets = .DEFAULT_PRESET) throws -> [String] {
+        var stringLen: Int32 = 0
+        var bytes = [CChar](repeating: 0, count: 4096)
+        let errorCode = get_eeg_names (boardId.rawValue, preset.rawValue, &bytes, &stringLen)
```

Comment:
not consistent code style, in some lines there are spaces before ( in some lines not, I am ok with both options in swift binding as long as its uniform. You can take one of the tools from this thread and apply it - https://forums.swift.org/t/swift-source-code-formatting-tool-like-clang-format-for-objective-c/416

Reply:
Do you mean the space between get_eeg_names and '('?  I prefer no space. I can remove the spaces if you prefer.

Reply:
yes, and its in plenty of places if you take a look carefully. Removing this manually is not good since you will waste more time doing it and more likely will miss smth. For C++ we use clang-format tool which formats code automatically, would be good to run formatter on the entire binding and force consistent code style this way

Reply:
like in the method below there is `getBoardDescr (_ ` and `get_board_descr (` and no space in other lines in the same method